<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Начать тепловой контроль</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1900</width>
        <height>2000</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2000</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacerL">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>660</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="StartTitle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>600</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Начать тепловой контроль</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="Startframe">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>600</width>
               <height>550</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QPushButton" name="StartStartButton">
              <property name="geometry">
               <rect>
                <x>436</x>
                <y>490</y>
                <width>120</width>
                <height>40</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Начать</string>
              </property>
             </widget>
             <widget class="QLabel" name="StartAuthorisationTitle">
              <property name="geometry">
               <rect>
                <x>48</x>
                <y>40</y>
                <width>508</width>
                <height>72</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24px; font-weight:600; color:#252525;&quot;&gt;Авторизация&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:14px; color:#252525;&quot;&gt;Введите ваше имя, фамилию и название объекта контроля&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="StartNameLineEdit">
              <property name="geometry">
               <rect>
                <x>196</x>
                <y>146</y>
                <width>360</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
             <widget class="QLineEdit" name="StartSurnameLineEdit">
              <property name="geometry">
               <rect>
                <x>196</x>
                <y>210</y>
                <width>360</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
             <widget class="QLineEdit" name="StartObjectLineEdit">
              <property name="geometry">
               <rect>
                <x>196</x>
                <y>274</y>
                <width>360</width>
               <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
             <widget class="QPushButton" name="StartExitButton">
              <property name="geometry">
               <rect>
                <x>306</x>
                <y>490</y>
                <width>120</width>
                <height>40</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Выход</string>
              </property>
             </widget>
             <widget class="QLabel" name="StartNameLabel">
              <property name="geometry">
               <rect>
                <x>48</x>
                <y>146</y>
                <width>110</width>
                <height>32</height>
               </rect>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Имя</string>
              </property>
             </widget>
             <widget class="QLabel" name="StartSurnameLabel">
              <property name="geometry">
               <rect>
                <x>48</x>
                <y>210</y>
                <width>110</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Фамилия</string>
              </property>
             </widget>
             <widget class="QLabel" name="StartObjectLabel">
              <property name="geometry">
               <rect>
                <x>48</x>
                <y>274</y>
                <width>110</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Объект контроля</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="StartPathLineEdit">
              <property name="geometry">
               <rect>
                <x>196</x>
                <y>338</y>
                <width>255</width>
                <height>32</height>
               </rect>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="StartPathLabel">
              <property name="geometry">
               <rect>
                <x>48</x>
                <y>338</y>
                <width>110</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Путь сохранения файлов</string>
              </property>
             </widget>
             <widget class="QPushButton" name="StartChangePathButton">
              <property name="geometry">
               <rect>
                <x>460</x>
                <y>338</y>
                <width>96</width>
                <height>32</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Изменить путь</string>
              </property>
             </widget>
             <zorder>StartNameLabel</zorder>
             <zorder>StartStartButton</zorder>
             <zorder>StartAuthorisationTitle</zorder>
             <zorder>StartNameLineEdit</zorder>
             <zorder>StartSurnameLineEdit</zorder>
             <zorder>StartObjectLineEdit</zorder>
             <zorder>StartExitButton</zorder>
             <zorder>StartSurnameLabel</zorder>
             <zorder>StartObjectLabel</zorder>
             <zorder>StartPathLineEdit</zorder>
             <zorder>StartPathLabel</zorder>
             <zorder>StartChangePathButton</zorder>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacerR">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>660</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res_rs.qrc"/>
 </resources>
 <connections/>
</ui>
